routes/cartRouter.js

```diff
@@ -0,0 +1,14 @@
+const express = require("express");
+const cartController = require("../controllers/cartController");
+
+const router = express.Router();
+
+// 유저 로그인 여부 검증
+const { validateToken } = require("../middlewares/auth.js");
+
+// 2. 장바구니 상품 조회
+router.get("/read", validateToken, cartController.readCart);
```

Comment:
- REST 규칙으로 URL을 만들 때는 동사를 사용하지 않습니다.
```suggestion
router.get("/", validateToken, cartController.readCart);
```